### DISCUSSION TITLE
Handle null value for bool type

### DISCUSSION THREAD

**Discussion by basigu** [microsoft/RulesEngine#406] [Q&A]
I have a property **scope** on an object in c#.  When **scope** is null the following expression should return true however it returns false.  What's the best way to handle nullable bool values?
{
        "RuleName": "HasIntegration",
        "Expression": true,
        "Actions": {
          "OnSuccess": {
            "Name": "OutputExpression",
            "Context": {
              "Expression": "input.Any(scope == true || scope == null))"
            }
          }
        }
      }

**Comment by YogeshPraj** [comment]
Hi @basigu ,

You can try this expression: 
"Expression": "input.Any(scope ?? true)"
